osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/monitoring/AbstractGetPowerQualityProfileHandler.java

```diff
@@ -267,12 +307,9 @@ private boolean isSelectedValue(final List<CaptureObjectDefinitionDto> selectedV
             return true;
         }
 
-        for (final CaptureObjectDefinitionDto captureObjectDefinition : selectedValues) {
-            if (this.isDefinitionOfSameObject(cosemObjectDefinitionDto, captureObjectDefinition)) {
-                return true;
-            }
-        }
-        return false;
+        return selectedValues.stream().anyMatch(
+                selectedValue -> this.isDefinitionOfSameObject(cosemObjectDefinitionDto, selectedValue));
+
     }
 
     private boolean isClockDefinition(final CosemObjectDefinitionDto cosemObjectDefinitionDto) {
```

Comment:
This method uses deprecated classes/methods.

Reply:
yes, the entire openmuc lib usage is deprecated, out of scope of this PR. needs investigation on how to proceed